python/settings/user_settings.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, bundle):
 
         self.__fw = sgtk.platform.current_bundle()
 
-        self.__settings = QtCore.QSettings("Shotgun Software", bundle.name)
+        self.__settings = QtCore.QSettings("ShotGrid Software", bundle.name)
```

Comment:
```suggestion
        self.__settings = QtCore.QSettings("Shotgun Software", bundle.name)
```

If we did this, then users would lose their settings.